tests/testing_utils.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import os
+import sys
+import inspect
+import logging
+import pytest
+
+from manim import logger
+from manim import dirs
+from manim import config
+
+
+class SceneTester:
+    """Class used to test the animations.
+
+    Parameters 
+    ----------
+    scene_object: : class:`Scene`
+        The scene to be tested
+    config_scene : :class:`Dict`
+        The configuration of the scene
+    module_tested: :class:`str`
+        The name of the module tested. i.e if we are testing functions of creation.py, the module will be "creation"
+
+    Attributes
+    -----------
+    path_tests_medias_cache : : class:`str`
+        Path to 'media' folder generated by manim. This folder contains cached data used by some tests. 
+    path_tests_data : : class:`str`
+        Path to the data used for the tests (i.e the pre-rendered frames).
+    scene : :class:`Scene`
+        The scene tested
```

Comment:
Won't work otherwise
```suggestion
    path_tests_medias_cache : :class:`str`
        Path to 'media' folder generated by manim. This folder contains cached data used by some tests. 
    path_tests_data : :class:`str`
        Path to the data used for the tests (i.e the pre-rendered frames).
    scene : :class:`Scene`
        The scene tested
```